main.js

```diff
@@ -181,6 +193,26 @@ async function _checkStatus({
     }
   }
 
+  /**
+   * This checks that the issuer of the verifiable credential matches
+   * the issuer of the revocationListCredential
+   */
```

Comment:
We use this comment style for simple comments on conditionals, short code blocks, etc:

```suggestion
  // this checks that the issuer of the verifiable credential matches
  // the issuer of the revocationListCredential
```

The other style is reserved for jsdocs/descriptions of classes, functions, etc.